documentation/sphinx/source/release-notes/release-notes-710.rst

```diff
@@ -11,6 +11,7 @@ Release Notes
 7.1.26
 ======
 * Released with AVX disabled.
+* Modified "status json" to not report fdbserver processes as clients. `(PR ##9252 <https://github.com/apple/foundationdb/pull/9252>)`_
```

Comment:
```suggestion
* Fixed status json to not report fdbserver processes as clients. `(PR #9252) <https://github.com/apple/foundationdb/pull/9252>`_
```

Reply:
Yes, compiling locally over that change.